tests/cases/compiler/omittedExpressionForOfLoop.ts

```diff
@@ -0,0 +1,11 @@
+for (const [,] of doesNotExist) {
+}
+
+for (const [,] of undefined) {
+}
+
+for (const [,] of []) {
+}
+
+for (const [] of []) {
```

Comment:
Can you add `// @strict: true` at the top of this test? These last two should be an error under `strictNullChecks`—without that enabled, we’re not really seeing if error reporting works on both.